index.html

```diff
@@ -16,6 +16,13 @@ <h1>Michi Berger</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  
+  <figure>
+    <img src="https://source.unsplash.com/random/600x300" alt="">
+    <figcaption>A random picture from unsplash.com</figcaption>
+  </figure>
+  
+
```

Comment:
There should not be more than one empty line, so please delete it before merging 👯 
```suggestion
```

Reply:
thx Paul...I´ve changed it now! I left just one empty line before and after `figure`